config/settings.yml.example

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
`Rack::Server` accepts `Host` and `Port` as options for the webserver and so I reused those names in the settings